### DISCUSSION TITLE
Skyfield, Moon apparent radius

### DISCUSSION THREAD

**Discussion by hidp123** [skyfielders/python-skyfield#764] [Q&A] [answered]
Hi, is it possible to obtain the apparent radius value for the moon. In the documentation an approximate of 0.25° has been given. Is there a way to calculate this more accurately?

JPL Horizons gives this value under the Ang-diam column.

Regards

**Comment by davidmikolas** [comment]
arctan(radius / distance) 

The physical radius can be looked up in Wikipedia (1737.4 km average of polar and equatorial for example), and the distance as a function of time can be calculated by Skyfield (usually between 360,000 and 410,000 km).

**Comment by hidp123** [comment, marked as answer]
https://rhodesmill.org/skyfield/examples.html#what-is-the-angular-diameter-of-a-planet-given-its-radius

One will have to supply the radius themselves though. JPL Horizons works it all out for you.
